src/machine/usb.go

```diff
@@ -123,6 +123,7 @@ var (
 	usbTxHandler    [usb.NumberOfEndpoints]func()
 	usbRxHandler    [usb.NumberOfEndpoints]func([]byte)
 	usbSetupHandler [usb.NumberOfInterfaces]func(usb.Setup) bool
+	usbStallHandler [usb.NumberOfEndpoints]func(usb.Setup) bool
```

Comment:
Does the stall handler really need the `usb.Setup` as well?

Reply:
Yep, I mentioned it in the novel I wrote just now, but the usbSetupHandler handles class-specific requests addressed to a protocol's interface, but stall handling is done via standard requests and addressed to the protocol endpoints individually